projects/ui/src/lib/components/po-info/po-info.component.spec.ts

```diff
@@ -159,6 +182,34 @@ describe('PoInfoComponent', () => {
       expect(nativeElement.querySelector('.po-text-nowrap')).toBeTruthy();
     });
 
+    it('should find `a.po-info-link` if `component.url` is truthy', () => {
+      component.value = 'John Doe';
+      component.url = 'http://portinari.io';
+
+      fixture.detectChanges();
+
+      expect(nativeElement.querySelector('a.po-info-link[target=_blank]')).toBeTruthy();
+    });
+
+    it('shouldn`t find `a.po-info-link` if `component.url` is truthy', () => {
```

Comment:
`shouldn't find 'a.po-info-link' if 'component.url' isn't truthy`, ou, `shouldn't find 'a.po-info-link' if 'component.url' is empty`, pois nesse caso URL não seria *truthy*.